packages/teams-js/test/public/people.spec.ts

```diff
@@ -26,270 +26,226 @@ describe('peoplePicker', () => {
       app._uninitialize();
     }
   });
+  const allowedContexts = [FrameContexts.content, FrameContexts.task, FrameContexts.settings];
+  describe('peoplePicker', () => {
+    /**
+     * People Picker tests
+     */
+    Object.values(FrameContexts).forEach(context => {
+      if (allowedContexts.some(allowedContext => allowedContext === context)) {
+        return;
+      }
+      it('should not allow selectPeople calls from the wrong context', async () => {
```

Comment:
Let's add the name of the context to the test the same way you did in the test after this. It's minor, but it can be a real help if a test fails to be able to tell exactly what the problem is just from reading the test name.